README.md

```diff
@@ -29,6 +29,13 @@ You can deploy impostman.el into your site-lisp as usual, then add this line to
 (require 'impostman)
 ```
 
+You can also use 'use-package' to install this package via MELPA as well:
+
+```elisp
+(use-package impostman
```

Comment:
I think it would be better to give instructions to install via `package-install` (no need to edit your startup file):

<kbd>M-x</kbd> `package-install` <kbd>RET</kbd> `impostman` <kbd>RET</kbd>